share/completions/rakudo.fish

```diff
@@ -0,0 +1,24 @@
+complete -c rakudo -n __fish_is_first_token -k -f -a '(__fish_complete_suffix)'
```

Comment:
This disables file completions via `-f`, only to reenable them via `__fish_complete_suffix`.

Also, `-k` (`--keep-order`) should not be used without a good reason. Please remove it!

Reply:
I think that's because I looked at [python.fish](https://github.com/fish-shell/fish-shell/blob/727934c6b6deb7a375531f3886e535eb4fc36d2a/share/completions/python.fish#L19).

Reply:
I guess that was more relevant when `__fish_complete_suffix .py` **only** showed `*.py` files.
Now it just sorts them first - this is why `-k` is needed. I guess it's still okay to keep